translation_en/chapter_02_PT.md

```diff
@@ -380,7 +380,7 @@ Isso baixa automaticamente a versão mais recente do stub e o executa antes que
 
 É importante observar que tanto o Serviço quanto os contratos são versionados juntos como parte da mesma base de código. Isso implica que o Stub gerado e o próprio Serviço terão a mesma versão. Um serviço de consumidor, para executar seus testes, pode depender do Stub, pois nunca deve depender do próprio serviço. Assim que o consumidor tiver testado usando o Stub de Serviço do produtor, você pode reconhecer rapidamente quando um contrato é quebrado ou quando uma nova versão do contrato não é mais compatível com os consumidores, pois os testes usando os Stubs serão interrompidos quando versões novas e incompatíveis forem lançadas. Neste momento, os consumidores se deparam com uma decisão simples: ficar dependendo dos contratos antigos com uma versão fixa, ou atualizar para a versão mais recente do contrato. Como você pode ver, isso pode exigir que você execute várias versões do seu serviço ao mesmo tempo. Felizmente para nós, o Kubernetes foi criado para oferecer suporte a esses cenários. Você pode ler sobre versões canário (Canary Releases) se estiver interessado em aspectos de multi-version deployments.
 
-> **DICA:** O capítulo [Cloud] () cobre Canary Releases, bem como outras estratégias de implantação. #TODO na fase de revisão: adicione referência a https://github.com/otaviojava/manual-arquiteto-moderno/blob/master/chapter_07.md
```

Comment:
Mesmo cenário:

Quando esse conteúdo for convertido a formato de livro online, esse link deveria ser uma ancora para o capitulo sete, não para o github. Será que seria possível esse tipo de ancora no conteúdo final?